_studio/mfx_lib/encode_hw/h265/include/mfx_h265_encode_hw_utils.h

```diff
@@ -325,6 +326,12 @@ struct RoiData{
     mfxU32  Bottom;
 
     mfxI16  Priority;
+
+    RoiData& operator = (mfxExtEncoderROI::roi roi)
+    {
+    	*this = roi;
```

Comment:
what is the point of operator = where you call operator = ?

Remove changes in this file